doc/examples/cleanup.sh

```diff
@@ -1,9 +1,11 @@
 #!/bin/bash -xe
 
-rm -rf */build
-rm -rf */android
-rm -rf */ios
-rm -rf */web
-rm -rf */macos
-rm -rf */test
-rm -rf */.dart_tool
+shopt -s globstar
```

Comment:
What is this?

Reply:
enables **

Reply:
Is this a bash thing? Works on Mac as well?

Reply:
it's bash. it should work on mac if it's compliant but wont work on sh/zsh/other shells I think. though i zsh ** works by default IIRC